jest.config.ts

```diff
@@ -124,11 +124,15 @@ export default {
     },
     {
       displayName: 'client-e2e',
-      testEnvironment: 'jsdom',
+      testEnvironment: 'node',
       preset: 'ts-jest',
       testMatch: ['<rootDir>/packages/client/integration/**/*.spec.ts'],
       modulePathIgnorePatterns: ['.*/node-modules/'],
       setupFiles: ['<rootDir>/packages/client/integration/step-definitions/setup.ts'],
+      moduleNameMapper: {
+        '^uuid$': require.resolve('uuid'),
```

Comment:
`uuid` doesn't ship a browser CJS module, unfortunately, so this is a workaround for that broken inport - it rewrites it to the node import.